### PR TITLE
Add gradle task for checking for aliases

### PR DESCRIPTION
**Pull request by supertassu**
Target Minecraft versions: any
Requirements: none
Related issues: https://github.com/SkriptLang/Skript/issues/1591#issuecomment-428318397

Description:
This PR adds a Gradle task to verify the existence of `skript-aliases/` to prevent `java.nio.file.NotDirectoryException: /aliases-english` when building the project without the alias submodule.

### CODE REVIEW

**Comment by Nicofisi** [comment]
I like the PR but I think there's one little thing that could be changed for it to be perfect - if you included the git command to initially grab the aliases and the command to update them later in the error message, that would be huge!

Also, the type of the exception probably doesn't change much here in how the error is displayed, but are you sure there's no better way of signaling the failure other than just throwing the generic `Exception` like you've done?

**Comment by supertassu** [comment]
> I think there's one little thing that could be changed for it to be perfect - if you included the git command to initially grab the aliases and the command to update them later in the error message, that would be huge!

I'll do that tomorrow.

> Also, the type of the exception probably doesn't change much here in how the error is displayed, but are you sure there's no better way of signaling the failure other than just throwing the generic Exception like you've done?

Atleast none that I know, however I have not done a lot of custom stuff.

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,31 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
```
[the remark this answers is not in the document]
Comment (supertassu) [reply .idea/misc.xml]: No idea. Will revert.

**Comment by Nicofisi** [comment]
Looks good, thanks for the contribution <3

**Pull request merged**